### PR TITLE
Added warning for gadgets when adding them from the store

### PR DESCRIPTION
**Pull request by rodrigopavezi**
modal.

This is a fix for #100.

This is how it looks like as we cannot test it on our stage environments due to lack of store data on rvacore-test.
screen shot 2015-10-28 at 2 15 10 pm

@ezequielc please review.

cheers

### CODE REVIEW

File: partials/playlist-item-modal.html [hunk 1]
```diff
@@ -7,6 +7,11 @@ <h3 class="modal-title" translate>editor-app.playlistItem.title</h3>
   </div>
   <div class="modal-body">
     <form role="form" name="presentationItemDetails" novalidate>
+    <div class="content-box" ng-show="item.type === 'Gadget'">
+      <div class="content-box-body bg-warning">
+        <p class="remove-bottom"><span translate>editor-app.playlistItem.legacyWarningPart1</span> <a  href="{{RVA_URL}}" target="_blank"><span translate>editor-app.playlistItem.legacyWarningPart2</span></a> <span translate>editor-app.playlistItem.legacyWarningPart3</span></p>
```
Comment (ezequielc) [inline partials/playlist-item-modal.html]: @rodrigopavezi Would it be hard to link to the current presentation in RVA? I did something similar in https://github.com/Rise-Vision/editor-app/blob/master/js/controllers/ctr-workspace.js#L39
I am not sure you have all the needed information in this scope. Cheers

**Comment by ezequielc** [comment]
@rodrigopavezi :+1: Just one comment. Otherwise looks good. Cheers

File: partials/playlist-item-modal.html (hunk 1 above)
Comment (rodrigopavezi) [inline partials/playlist-item-modal.html]: @ezequielc good point. Just adding this. cheers

**Comment by rodrigopavezi** [comment]
@ezequielc I have changed the url, please review. cheers

**Comment by ezequielc** [comment]
@rodrigopavezi Nice :+1:

**Pull request merged**